recipes/ssht/all/conanfile.py

```diff
@@ -1,9 +1,9 @@
-from conans import CMake, ConanFile, tools
-from conans.errors import ConanInvalidConfiguration
-from glob import glob
-import os
-
+from conan import ConanFile
+from conan.tools import files
+from conan.errors import ConanInvalidConfiguration
+from conans import CMake
 
+required_conan_version = ">=1.41.0"
```

Comment:
```suggestion
required_conan_version = ">=1.50.0"
```
conan.errors comes from 1.50

Reply:
that's not whathttps://github.com/conan-io/conan-center-index/blob/master/docs/v2_linter.md says :/